charts/ceph-csi-cephfs/templates/provisioner-statefulset.yaml

```diff
@@ -104,16 +104,17 @@ spec:
               mountPath: /tmp/csi/keys
           resources:
 {{ toYaml .Values.nodeplugin.plugin.resources | indent 12 }}
+{{- if .Values.provisioner.httpMetrics.enabled }}
         - name: liveness-prometheus
           image: "{{ .Values.nodeplugin.plugin.image.repository }}:{{ .Values.nodeplugin.plugin.image.tag }}"
+          imagePullPolicy: {{ .Values.nodeplugin.plugin.image.pullPolicy }}
           args:
             - "--type=liveness"
             - "--endpoint=$(CSI_ENDPOINT)"
-            - "--metricsport=8081"
-            - "--metricspath=/metrics"
+            - "--livenessport={{ .Values.provisioner.httpMetrics.containerPort }}"
```

Comment:
livenessport and livenesspath replaced by mertricsport and metricspath

Reply:
Good catch, such a big refactor and at some point you become blind to your own mistakes ;)